test/analytics/tests/instrumented_events/from_the_browser/core_context_providers.ts

```diff
@@ -17,10 +16,10 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
   const { common } = getPageObjects(['common']);
 
   describe('Core Context Providers', () => {
-    let event: Event;
+    let event: Event<Record<string, unknown>>;
     before(async () => {
       await common.navigateToApp('home');
-      [event] = await ebtUIHelper.getEvents(1, { eventTypes: [KIBANA_LOADED_EVENT] }); // Get the loaded Kibana event
+      [event] = await ebtUIHelper.getEvents(1, { eventTypes: ['Loaded Kibana'] }); // Get the loaded Kibana event
```

Comment:
We were importing a different constant. But also, I'd rather validate the expected behaviour instead of relying on a variable that may change (hence breaking the contract).